site-example.yml

```diff
@@ -24,7 +29,8 @@ Documents:                                                              # Requir
         SEO Description: "Description"                                  # Optional - Meta description tag for SEO
         SEO Keywords: "keyword1, keyword2, keyword3"                    # Optional - Meta keywords tag for SEO
         Options: hidden                                                 # Optional - The value 'hidden' allows user to render page but hide sidebar link
-
+        OtherLanguages:                                                 # Optional - Available Translations
+          - cn
```

Comment:
@kanagarajkm do we really need this in the site.yml file? Can we infer this based on the original Language settings.

Reply:
@deekoder yes, we could have avoided it. But some of the docs which are in english does not have equivalent chinese docs. We might end up with lot of 404 links.

Reply:
I see. Ok @kaankabalak explained that to me. So that's why it's at a document level.